### ISSUE TITLE
problem adding an addon

### ISSUE THREAD

**Issue by martialgallorini** [ofZach/projectGeneratorSimple#68]
Hello,

Syphon is really a must have for openFrameworks.
Unfortunately, ofxSyphon addon is quite complicated to use for XCode noobs.

I tried and keep trying but can't get it to work with oF0.8.4

Is there a chance this addons will work properly with projectGenerator one day ?

thanks a lot by the way

**Comment by bangnoise** [comment]
Long-standing unresolved bugs blocking this are #56 and #58.

**Comment by martialgallorini** [comment]
yeah i saw that. But that's why you provided some workaround i suppose.

but unfortunately i can't get it to work even after applying your suggestions.

Maybe i am doing things wrong

i filed project generator developper a bug, hope it helps, but things doesn't sem to move over there...

**Comment by ofZach** [comment]
thanks, this should be fixed by 0.9

**Comment by martialgallorini** [comment]
Great News ! thanks i will check upcoming version.

Will it make it in the nightly builds ?

**Comment by martialgallorini** [comment]
Hello,

Any update on this bug ?

thanks
